pkg/core/table/locking.go

```diff
@@ -548,13 +619,11 @@ func LoadPBAImage(s *core.Session, image []byte) error {
 	return nil
 }
 
-func RevertLockingSP(s *core.Session, keep bool, pwhash []byte) error {
+func RevertLockingSP(s *core.Session, keep bool) error {
 	mc := method.NewMethodCall(uid.InvokeIDThisSP, uid.OpalRevertSP, s.MethodFlags)
 	if keep {
 		mc.Token(stream.StartName)
-		// KeepGlobalRangeKey, TCG Storage Security Subsystem Class: Opal | Version 2.02 | Revision 1.0 | Page 85
-		// sedutil-cli looks like a Short-Atom without byte or integer indicator
-		mc.RawByte([]byte{0x83, 0x06, 0x00, 0x00})
+		mc.UInt(KeepGlobalRangeKey)
```

Comment:
Im not sure, but after checking, this will change `[]byte{0x83, 0x06, 0x00, 0x00}` to `[]byte{0x84, 0x06, 0x00, 0x00}`

Does it work the same?

Reply:
yes It should, It's just a number according to the spec and sedutil decided to use a 3 byte integer for it. Let me double check it on some drive to be 100% sure